naming/src/main/java/com/alibaba/nacos/naming/misc/HttpClient.java

```diff
@@ -103,7 +102,13 @@ public static HttpResult request(String url, List<String> headers, Map<String, S
             String encodedContent = encodingParams(paramValues, encoding);
             url += (null == encodedContent) ? "" : ("?" + encodedContent);
 
-            conn = (HttpURLConnection) new URL(url).openConnection();
+            if(CON_PROXY_ENABLED){
```

Comment:
This HttpClient is just for naming module, can you add the same proxy setting for config module?